docs/getting-started.md

```diff
@@ -7,28 +7,126 @@ title: Getting Started
 De 'WOZ Bevragen' API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
 
 Om aan te sluiten kun je de volgende stappen doorlopen:
-1. [Bekijk de functionaliteit en specificaties](#functionaliteit-en-specificaties)
-2. [Implementeer de API](#bouw-de-api)
-3. [Probeer en test de API](#probeer-en-test-de-api)
+1. [Meld je aan bij het kadaster om toegang te krijgen](#aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#functionaliteit-en-specificaties)
+3. [Implementeer de API](#bouw-de-api)
+4. [Probeer en test de API](#probeer-en-test-de-api)
+5. [Sluit aan op productie](#aansluiten-op-productie)
+
+## Aanmelden om aan te sluiten
+Meld je aan bij het kadaster om [aan te sluiten en voor toegang tot de testomgeving](???????????????????????????). Je ontvangt dan o.a. een API-key die nodig is voor toegang tot de testomgeving.
```

Comment:
lijkt me dat dit goed is. Ik heb Kallisti gevraagd of we alleen een API key voor de testomgeving krijgen of tegelijkertijd ook voor productie en evt instructies voor het implementeren van dubbelzijdig TLS? Moet je voor dat laatste een nieuwe aanmelding doen? etc. we wachten het even af.

Reply:
Heb gezien dat je op de site van het kadaster moet aangeven of het om een productie API key gaat, of om een key voor de ETO. Het is echter hetzelfde formulier...